docs/acsengine.md

```diff
@@ -52,105 +129,50 @@ Use "acs-engine [command] --help" for more information about a command.
 
 [Here's a quick demo video showing the dev/build/test cycle with this setup.](https://www.youtube.com/watch?v=lc6UZmqxQMs)
 
-# Downloading and Building ACS Engine Locally
+## Building on Windows, OSX, and Linux
 
-ACS Engine can also be built and run natively on Windows, OS X, and Linux. Instructions below:
+Building ACS Engine from source has a few requirements for each of the platforms. Download and install the pre-reqs for your platform, Windows, Linux, or Mac:
 
-## Windows
+1. Go version 1.8 [installation instructions](https://golang.org/doc/install)
+2. Git Version Control [installation instructions](https://git-scm.com/download/)
 
-Requirements:
-- Git for Windows. Download and install [here](https://git-scm.com/download/win)
-- Go for Windows. Download and install [here](https://golang.org/dl/), accept all defaults.
-- Powershell
+### Windows
```

Comment:
Probably for another PR, but it would be great to get everyone to build using the scripts/devenv.* scripts.  That way everyone works from the same docker container, and is consistent across all OS's.